src/routes/secondRouter.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import * as secondController from '../controllers/secondController';
```

Comment:
same here
you must adhere to a single format 
if you use import { Router } from 'express'; please do the same for secondController functions